From 8ee1247da3de151fc3e01518b261d962c8a3418c Mon Sep 17 00:00:00 2001
From: Igor Grinberg <igrinberg@habana.ai>
Date: Wed, 22 Sep 2021 19:25:05 +0300
Subject: [PATCH] [SW-55215] gaudi: use the new cold rst data name

Specs has switched to a new name which better reflects the data stored
and not the storage used.
Switch to new name.

Change-Id: Ib71fa91f1815a14b5ab822ab2d3b654c12b53aad
Signed-off-by: Igor Grinberg <igrinberg@habana.ai>
---

diff --git a/arch/gaudi/arm/common.c b/arch/gaudi/arm/common.c
index a769327..55f145b 100644
--- a/arch/gaudi/arm/common.c
+++ b/arch/gaudi/arm/common.c
@@ -505,17 +505,17 @@
 void preboot_sp_sram_init(void)
 {
 	register int i;
-	struct gaudi_flops_2_data flops_2_data;
+	struct gaudi_cold_rst_data cold_rst_data;
 	u32 *data = (u32 *)(HL_SP_SRAM_BASE);
 
-	flops_2_data.data = regmap_readl(mmCOLD_RST_DATA);
-	if (flops_2_data.spsram_init_done != SPSRAM_INIT_DONE) {
+	cold_rst_data.data = regmap_readl(mmCOLD_RST_DATA);
+	if (cold_rst_data.spsram_init_done != SPSRAM_INIT_DONE) {
 		/* SP SRAM access is limited to 32bit */
 		for (i = 0; i < HL_SP_SRAM_SIZE / sizeof(u32); i++)
 			data[i] = RESET_PATTERN;
 
-		flops_2_data.spsram_init_done = SPSRAM_INIT_DONE;
-		regmap_writel(flops_2_data.data, mmCOLD_RST_DATA);
+		cold_rst_data.spsram_init_done = SPSRAM_INIT_DONE;
+		regmap_writel(cold_rst_data.data, mmCOLD_RST_DATA);
 	}
 
 	cpu_boot_dev_sts_clrset(CPU_BOOT_DEV_STS0_SP_SRAM_EN, true);
